gatsby-node.js

```diff
@@ -124,7 +113,10 @@ exports.createPages = async ({ actions }) => {
             );
             const slug = getPageSlug(page);
             if (pageNodes.query_fields) {
-                const relatedPages = getRelatedPagesWithImages(pageNodes);
+                const relatedPages = getRelatedPagesWithImages(
+                    pageNodes,
+                    RESOLVED_REF_DOC_MAPPING
```

Comment:
nit: we can probably camelCase this variable now - there's nothing `constant` about it.